src/terrain/terrain.js

```diff
@@ -380,7 +398,8 @@ export class Terrain extends Elevation {
         }
 
         const options = this.painter.options;
-        this.drapeFirst = (options.zooming || options.moving || options.rotating || !!this.forceDrapeFirst) && !this._invalidateRenderCache;
+        this.renderCached = (options.zooming || options.moving || options.rotating || !!this.forceRenderCached) && !this._invalidateRenderCache;
```

Comment:
We could remove check against options.zooming || options.moving || options.rotating. I opted to leave as is in this PR, invalidation of cache will need to happen on video update event and feature state to make that work properly. I'll create a follow-up ticket for this.